maquetado/scss/login.scss

```diff
@@ -0,0 +1,44 @@
+@import 'colors';
+
+.login {
+  background-color: $wild-sand;
+  border-top: solid 8px $cerulean;
+  display: flex;
+  flex-direction: column;
+  height: 400px;
+  justify-content: space-between;
+  margin: 180px auto;
+  padding: 25px;
+  width: 300px;
+}
+
+.text-from {
```

Comment:
I think this styles already exits, so don't repeat, create a file for text something like text.scss and import it when do you need